src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -5,7 +5,7 @@ import org.broadinstitute.dsde.agora.server.AgoraConfig
 import org.broadinstitute.dsde.agora.server.dataaccess.AgoraDao
 import org.broadinstitute.dsde.agora.server.model.{AgoraEntity, AgoraEntityProjection, AgoraEntityType}
 
-class AgoraBusiness(authorizationProvider: AuthorizationProvider) {
+class AgoraBusiness {
```

Comment:
Is there still value in having this layer, if all it seems to add over the DAO is the URL?  More a question for George/Jay when they get back. Not a problem for this pull I think